Makefile

```diff
@@ -75,13 +76,17 @@ build-darwin-amd64: ## Create the etcdproxy-controller executable for Darwin (os
 build-windows-amd64: ## Create the etcdproxy-controller executable for Windows 64-bit OS in the ./bin directory. Requires Docker.
 	GOOS=windows GOARCH=amd64${GOBUILD} -o bin/windows-amd64 &
 
-.PHONY: test
-test: ## Run tests.
-	go test -timeout 20m -v $(PKGS)
+.PHONY: test-ci
+test-ci: ## Run tests.
+	go test -timeout 20m -v $(CI_PKGS)
 
-.PHONY: verify-ci
-verify-ci: install-tools ## Run code checks
-	PKGS="${GOFILES}" GOFMT="gofmt" ./hack/verify-ci.sh
+.PHONY: test-e2e
+test-e2e: ## Run E2E tests. Requires working Kubernetes cluster and a Kubeconfig file.
```

Comment:
This should be updated to use the `run-e2e.sh` script.

A follow-up question: do we need to leave a note that this is potentially destructive operation and to be sure you're running it on the correct cluster?

Reply:
> A follow-up question: do we need to leave a note that this is potentially destructive operation and to be sure you're running it on the correct cluster?

note that your tests may be destructive.  We can assume these people know what they're doing and get what they deserve.

Reply:
Added a note and fixed the command.